src/py/mat3ra/made/tools/README.md

```diff
@@ -0,0 +1,178 @@
+- Configurations.
+- Factories: methods that manage the creation of different types of builders.
+
+
+### 2.2. Top-level Entrypoint Functions.
+
+For usability purposes, we provide top-level functions that allow for the creation of materials with a single call. These functions are:
+
+[//]: # (Explain the helper functions and their purpose `create_interface`.)
+
+The function with plural name takes in Configuration and Builder as arguments and returns the Result Materials list when multiple results are possible.
+
+- `create_interfaces`
+- `create_defects`
+
+
+The function with singular name takes in Configuration and Builder as arguments and returns single Result Material object when only one result is possible, or the first, most optimal one, from the list of results ordered by some criteria.
+
+- `create_interface`
+- `create_defect`
+- `create_slab_defect`
+- `create_slab`
+- `create_perturbation`
+- `create_passivation`
+- `create_nanoribbon`
+- `create_grain_boundary`
+
+Some of the top-level functions are helpers that allow to get additional properties from configuration to refine the build process, such as:
+- `get_terminations` for slab
+- `get_unique_coordination_numbers` for passivation
+- `get_optimal_film_displacement` for interface
+
+
+## 3. Usage
+
+### 3.1. Installation
+
+See top-level [README](LINK) for installation instructions.
+
+### 3.2. Usage in Python
+
+Here is an example workflow using made.tools.build:
+
+Define Configuration:
+
+```python
+from mat3ra.made.tools.build.perturbation.configuration import PerturbationConfiguration
+from mat3ra.made.tools.build.perturbation.builders import SlabPerturbationBuilder
+
+perturbation_config = PerturbationConfiguration(
+    material=initial_material,
+    perturbation_function_holder=perturbation_function_holder,
+    use_cartesian_coordinates=False
+)
+```
+
+then, create Builder and generate Material:
+
+```python
+builder = SlabPerturbationBuilder()
+perturbed_slab = builder.get_material(perturbation_config)
+```
+
+or, alternatively, use the top-level function:
+
+```python
+from mat3ra.made.tools import create_perturbation
+
+perturbed_slab = create_perturbation(perturbation_config,
+                                     builder=SlabPerturbationBuilder)
+```
+
+Finally, one can review the content of the material:
+
+```python
+print(perturbed_slab.to_json())
+```
+
+
+
+## Links
+
+1. [Exabyte Source of Schemas and Examples (ESSE), Github Repository](https://github.com/exabyte-io/exabyte-esse)
+2. [ECMAScript 2015 Language Specifications](https://www.ecma-international.org/ecma-262/6.0/)
+3. [POSCAR file format, official website](https://cms.mpi.univie.ac.at/vasp/guide/node59.html)
+4. [XYZ file format, Wikipedia](https://en.wikipedia.org/wiki/XYZ_file_format)
+5. [Quantum ESPRESSO, Official Website](https://www.quantum-espresso.org/)
+6. [Mocha, Official Website](https://mochajs.org/)
+7. [Git LFS, Official Website](https://git-lfs.github.com/)
```

Comment:
We need to update these links
- ASE
- Pymatgen
- https://arxiv.org/abs/1902.10838, ESSE github